llvm/test/Other/dump-before-after.ll

```diff
@@ -0,0 +1,57 @@
+; RUN: mkdir -p %t/logs
```

Comment:
Ensure that the test could be run on Windows host

Reply:
this appears in lots of tests, it should work on windows

Reply:
> this appears in lots of tests, it should work on windows

yeah, indeed. Are we sure about find / sort though?

Reply:
It passes the windows CI on this PR, is that assurance enough?